src/pal/src/arch/arm/exceptionhelper.S

```diff
@@ -22,12 +22,7 @@ LEAF_ENTRY ThrowExceptionFromContextInternal, _TEXT
     ldr	r10,	[r0, #(CONTEXT_R10)]
     ldr	r11,	[r0, #(CONTEXT_R11)]
     ldr	sp,	[r0, #(CONTEXT_Sp)]
-
-    ldr	r2,	[r0, #(CONTEXT_Pc)]
-
-    // Store return address to the stack
-    // Added r7 as a dummy to keep the stack aligned by 8 bytes.
-    push    {r2, r7}
+    ldr	lr,	[r0, #(CONTEXT_Pc)]
 
```

Comment:
Yes. We are returning to this context (r0, the first argument).
Note that the twin code of x86 has
```
mov rax, [rdi + CONTEXT_Rip]
push_register rax
```
, which is translated as
"Save first_argument->PC(IP) to the return address location in the stack",
which is translated as
```
ldr lr, [r0, #(CONTEXT_Pc)]
```
for ARM.